### DISCUSSION TITLE
Key Vault Access

### DISCUSSION THREAD

**Discussion by GeorgeAtMOD** [thunderclient/thunder-client-support#557] [Q&A]
I am rather new to Thunder Client but so far I am liking it.   I want to have your extension access our Key Vault so that I can set a variable to the necessary secret values.  Otherwise I'll have to keep the secrets in our code which makes my inner security monkey scream and how.

I am a complete newbie in this regard so any pointers or suggestions or how-to articles would be a great help.

**Comment by rangav** [comment]
Thanks @GeorgeAtMOD for the suggestion,

Is this how you access key vault in Node?
https://docs.microsoft.com/en-us/azure/key-vault/secrets/quick-create-node

**Reply by GeorgeAtMOD** [reply]
@rangav, thank you for the quick response.

I am reading through the article but I may be missing a mental leap of some sort.  My goal is to use a token saved in keyvault as a variable, such as {{myToken}}, and use that variable in various post or get requests.

**Reply by rangav** [reply]
Thanks, will look into it.

also will wait for other users feedback on this.

**Reply by GeorgeAtMOD** [reply]
You know, I misunderstood what you were originally asking me.  We use a similar process to access our key vault from within databricks or azure data factory.  What I do not know how to do is to access the key vault within the Thunder Client extension.

Thanks again!
